// @salesforce-ux/design-system/design-tokens/dist/bg-standard.common.js
module.exports = {
  contact: "rgb(160, 148, 237)",
  multiSelectCheckbox: "rgb(150, 148, 146)",
  workOrder: "rgb(80, 227, 194)",
  post: "rgb(101, 202, 228)",
  globalConstant: "rgb(84, 105, 141)",
  carousel: "rgb(107, 189, 110)",
  workContract: "rgb(0, 161, 224)",
  resourceSkill: "rgb(69, 193, 115)",
  systemAndGlobalVariable: "rgb(84, 105, 141)",
  segments: "rgb(242, 139, 0)",
  goals: "rgb(86, 170, 223)",
  adjustValue: "rgb(71, 180, 108)",
  caseWrapUp: "rgb(242, 207, 91)",
  investmentAccount: "rgb(75, 192, 118)",
  store: "rgb(4, 132, 75)",
  slider: "rgb(150, 148, 146)",
  output: "rgb(67, 156, 186)",
  medicationIngredient: "rgb(52, 190, 205)",
  storeGroup: "rgb(60, 151, 221)",
  all: "rgb(84, 105, 141)",
  serviceTerritoryPolicy: "rgb(21, 137, 238)",
  picklistChoice: "rgb(84, 105, 141)",
  choice: "rgb(84, 105, 141)",
  app: "rgb(252, 185, 91)",
  default: "rgb(129, 153, 175)",
  caseMilestone: "rgb(242, 207, 91)",
  today: "rgb(239, 126, 173)",
  buyerAccount: "rgb(4, 132, 75)",
  leadList: "rgb(248, 137, 98)",
  workPlanTemplate: "rgb(68, 148, 136)",
  shift: "rgb(235, 112, 146)",
  productItemTransaction: "rgb(248, 137, 98)",
  apex: "rgb(84, 105, 141)",
  replyText: "rgb(248, 137, 101)",
  answerPrivate: "rgb(242, 207, 91)",
  immunization: "rgb(235, 112, 146)",
  assetDowntimePeriod: "rgb(49, 122, 146)",
  opportunityContactRole: "rgb(126, 139, 228)",
  retailBankingConsole: "rgb(0, 175, 160)",
  channelProgramMembers: "rgb(14, 181, 138)",
  appsAdmin: "rgb(152, 149, 238)",
  datadotcom: "rgb(21, 137, 238)",
  settings: "rgb(4, 132, 75)",
  productItem: "rgb(118, 158, 217)",
  metrics: "rgb(86, 170, 223)",
  topic2: "rgb(86, 170, 208)",
  partnerFundAllocation: "rgb(14, 181, 138)",
  careRequestReviewer: "rgb(75, 192, 118)",
  approval: "rgb(80, 204, 122)",
  workQueue: "rgb(84, 105, 141)",
  iotOrchestrations: "rgb(42, 115, 158)",
  visualforcePage: "rgb(252, 185, 91)",
  personAccount: "rgb(127, 141, 225)",
  entity: "rgb(248, 137, 98)",
  serviceTerritoryLocation: "rgb(126, 139, 228)",
  entitlementPolicy: "rgb(4, 132, 75)",
  form: "rgb(110, 192, 110)",
  products: "rgb(4, 132, 75)",
  orderItem: "rgb(118, 158, 211)",
  readReceipts: "rgb(75, 192, 118)",
  javascriptButton: "rgb(252, 185, 91)",
  maintenanceAsset: "rgb(42, 115, 158)",
  workPlanRule: "rgb(68, 148, 136)",
  loop: "rgb(255, 154, 60)",
  portalRolesAndSubordinates: "rgb(122, 154, 230)",
  workCapacityLimit: "rgb(0, 121, 188)",
  employeeOrganization: "rgb(0, 161, 224)",
  constant: "rgb(84, 105, 141)",
  marketingActions: "rgb(107, 189, 110)",
  caseTranscript: "rgb(242, 207, 91)",
  timesheetEntry: "rgb(125, 195, 125)",
  keyDates: "rgb(107, 158, 226)",
  multiPicklist: "rgb(150, 148, 146)",
  productWorkspace: "rgb(4, 132, 75)",
  visitTemplates: "rgb(60, 151, 221)",
  restrictionPolicy: "rgb(21, 137, 238)",
  task: "rgb(75, 192, 118)",
  answerBest: "rgb(242, 207, 91)",
  assetAction: "rgb(49, 122, 146)",
  section: "rgb(150, 148, 146)",
  serviceRequest: "rgb(21, 137, 228)",
  orders: "rgb(118, 158, 217)",
  propagationPolicy: "rgb(21, 137, 238)",
  pastChat: "rgb(248, 137, 96)",
  feedback: "rgb(109, 161, 234)",
  actionListComponent: "rgb(88, 118, 163)",
  opportunitySplits: "rgb(252, 185, 91)",
  locationPermit: "rgb(52, 181, 157)",
  messagingUser: "rgb(52, 190, 205)",
  filterCriteria: "rgb(42, 115, 158)",
  trailhead: "rgb(3, 46, 97)",
  entitlements: "rgb(183, 129, 211)",
  checkout: "rgb(4, 132, 75)",
  formula: "rgb(84, 105, 141)",
  caseLogACall: "rgb(242, 207, 91)",
  shiftPatternEntry: "rgb(235, 112, 146)",
  thanksLoading: "rgb(184, 195, 206)",
  jobFamily: "rgb(0, 161, 224)",
  serviceAppointmentCapacityUsage: "rgb(0, 121, 188)",
  channelProgramLevels: "rgb(14, 181, 138)",
  letterhead: "rgb(60, 151, 221)",
  emailChatter: "rgb(242, 207, 91)",
  documentReference: "rgb(42, 115, 158)",
  announcement: "rgb(98, 183, 237)",
  bot: "rgb(84, 105, 143)",
  macros: "rgb(71, 207, 210)",
  dashboardEa: "rgb(126, 139, 228)",
  jobProfile: "rgb(235, 112, 146)",
  steps: "rgb(84, 105, 141)",
  assetRelationship: "rgb(250, 151, 92)",
  highVelocitySales: "rgb(71, 207, 201)",
  brand: "rgb(126, 139, 228)",
  visits: "rgb(60, 151, 221)",
  einsteinReplies: "rgb(248, 137, 101)",
  coaching: "rgb(246, 117, 148)",
  recordLookup: "rgb(235, 112, 146)",
  productServiceCampaignItem: "rgb(42, 115, 158)",
  schedulingConstraint: "rgb(235, 112, 146)",
  lightningComponent: "rgb(150, 148, 146)",
  search: "rgb(98, 183, 237)",
  connectedApps: "rgb(152, 149, 238)",
  swarmRequest: "rgb(226, 110, 140)",
  workTypeGroup: "rgb(0, 121, 188)",
  education: "rgb(60, 151, 221)",
  workType: "rgb(84, 105, 141)",
  travelMode: "rgb(69, 193, 115)",
  environmentHub: "rgb(84, 105, 141)",
  cms: "rgb(136, 198, 81)",
  callCoaching: "rgb(60, 151, 221)",
  salesforceCms: "rgb(0, 161, 223)",
  thanks: "rgb(233, 105, 110)",
  serviceTerritoryMember: "rgb(126, 139, 228)",
  workForecast: "rgb(252, 185, 91)",
  holidayOperatingHours: "rgb(107, 158, 226)",
  userRole: "rgb(143, 201, 110)",
  observationComponent: "rgb(127, 141, 225)",
  orchestrator: "rgb(0, 110, 179)",
  workCapacityUsage: "rgb(0, 121, 188)",
  recordCreate: "rgb(235, 112, 146)",
  campaignMembers: "rgb(244, 151, 86)",
  retailBanking: "rgb(0, 210, 190)",
  productServiceCampaign: "rgb(42, 115, 158)",
  portalRoles: "rgb(251, 137, 80)",
  calibration: "rgb(71, 207, 210)",
  answerPublic: "rgb(242, 207, 91)",
  displayText: "rgb(150, 148, 146)",
  unmatched: "rgb(98, 183, 237)",
  partners: "rgb(14, 181, 138)",
  emailIq: "rgb(160, 148, 237)",
  serviceCrew: "rgb(250, 151, 92)",
  paymentGateway: "rgb(11, 130, 124)",
  voiceCall: "rgb(48, 200, 90)",
  cancelCheckout: "rgb(150, 148, 146)",
  assetWarranty: "rgb(84, 105, 141)",
  resourceCapacity: "rgb(69, 193, 115)",
  channelPrograms: "rgb(14, 181, 138)",
  quip: "rgb(211, 69, 29)",
  quipSheet: "rgb(48, 200, 90)",
  timeslot: "rgb(250, 178, 76)",
  liveChat: "rgb(248, 137, 96)",
  serviceRequestDetail: "rgb(52, 190, 205)",
  jobPosition: "rgb(0, 161, 224)",
  sobjectCollection: "rgb(84, 105, 141)",
  user: "rgb(52, 190, 205)",
  client: "rgb(0, 210, 190)",
  screen: "rgb(21, 137, 238)",
  portal: "rgb(174, 199, 112)",
  partnerFundRequest: "rgb(14, 181, 138)",
  resourcePreference: "rgb(69, 193, 115)",
  firstNonEmpty: "rgb(233, 105, 110)",
  customer360: "rgb(3, 46, 97)",
  employeeJob: "rgb(0, 161, 224)",
  resourceAbsence: "rgb(69, 193, 115)",
  textTemplate: "rgb(84, 105, 141)",
  entitlementTemplate: "rgb(126, 139, 228)",
  lightningUsage: "rgb(126, 139, 228)",
  entitlement: "rgb(126, 139, 228)",
  empty: "rgb(129, 153, 175)",
  text: "rgb(150, 148, 146)",
  delegatedAccount: "rgb(4, 132, 75)",
  fulfillmentOrder: "rgb(185, 172, 145)",
  caseEmail: "rgb(242, 207, 91)",
  account: "rgb(127, 141, 225)",
  assignment: "rgb(255, 154, 60)",
  wealthManagementConsole: "rgb(0, 175, 160)",
  task2: "rgb(75, 192, 118)",
  codePlayground: "rgb(84, 105, 141)",
  social: "rgb(234, 116, 162)",
  endorsement: "rgb(139, 154, 227)",
  folder: "rgb(139, 154, 227)",
  serviceCrewMember: "rgb(126, 139, 228)",
  variationAttributeSetup: "rgb(4, 132, 75)",
  flow: "rgb(0, 121, 188)",
  recipe: "rgb(1, 118, 211)",
  expenseReportEntry: "rgb(58, 49, 128)",
  employee: "rgb(0, 161, 224)",
  omniSupervisor: "rgb(138, 118, 240)",
  capacityPlan: "rgb(235, 112, 146)",
  assetObject: "rgb(49, 122, 146)",
  product: "rgb(183, 129, 211)",
  topic: "rgb(86, 170, 223)",
  productRequired: "rgb(239, 110, 100)",
  dynamicRecordChoice: "rgb(84, 105, 141)",
  aggregationPolicy: "rgb(21, 137, 238)",
  warrantyTerm: "rgb(0, 121, 188)",
  process: "rgb(0, 121, 188)",
  swarmSession: "rgb(245, 103, 91)",
  people: "rgb(52, 190, 205)",
  reward: "rgb(233, 105, 110)",
  employeeContact: "rgb(0, 161, 224)",
  bundlePolicy: "rgb(21, 137, 238)",
  performance: "rgb(248, 177, 86)",
  promotions: "rgb(4, 132, 75)",
  caseComment: "rgb(242, 207, 91)",
  priceBookEntries: "rgb(4, 132, 75)",
  salesChannel: "rgb(42, 115, 158)",
  apexPlugin: "rgb(84, 105, 141)",
  campaign: "rgb(244, 151, 86)",
  contactRequest: "rgb(251, 137, 84)",
  businessHours: "rgb(125, 195, 125)",
  evernote: "rgb(134, 200, 111)",
  variationProducts: "rgb(4, 132, 75)",
  trailheadAlt: "rgb(0, 57, 107)",
  serviceTerritory: "rgb(126, 139, 228)",
  customerLifecycleAnalytics: "rgb(146, 116, 223)",
  case: "rgb(242, 207, 91)",
  currency: "rgb(150, 148, 146)",
  record: "rgb(125, 195, 125)",
  queue: "rgb(84, 105, 141)",
  scheduleObjective: "rgb(42, 115, 158)",
  contractLineItem: "rgb(110, 192, 110)",
  skillEntity: "rgb(139, 154, 227)",
  skill: "rgb(250, 151, 92)",
  operatingHours: "rgb(107, 158, 226)",
  custom: "rgb(129, 153, 175)",
  relatedList: "rgb(89, 188, 171)",
  botTraining: "rgb(88, 118, 163)",
  caseChangeStatus: "rgb(242, 207, 91)",
  insuranceConsole: "rgb(0, 175, 160)",
  contract: "rgb(110, 192, 110)",
  sobject: "rgb(150, 148, 146)",
  salesCadenceTarget: "rgb(84, 105, 141)",
  photo: "rgb(215, 209, 209)",
  apps: "rgb(60, 151, 221)",
  timesheet: "rgb(126, 139, 228)",
  drafts: "rgb(108, 161, 233)",
  outcome: "rgb(255, 154, 60)",
  workOrderItem: "rgb(51, 168, 220)",
  pricebook: "rgb(183, 129, 211)",
  scanCard: "rgb(243, 158, 88)",
  note: "rgb(230, 212, 120)",
  serializedProductTransaction: "rgb(42, 115, 158)",
  shiftPattern: "rgb(235, 112, 146)",
  buyerGroup: "rgb(4, 132, 75)",
  productQuantityRules: "rgb(4, 132, 75)",
  procedure: "rgb(21, 137, 228)",
  opportunity: "rgb(252, 185, 91)",
  news: "rgb(127, 141, 225)",
  sortPolicy: "rgb(21, 137, 238)",
  displayRichText: "rgb(150, 148, 146)",
  strategy: "rgb(75, 192, 113)",
  callHistory: "rgb(242, 207, 91)",
  webcart: "rgb(4, 132, 75)",
  serializedProduct: "rgb(42, 115, 158)",
  report: "rgb(46, 203, 190)",
  groups: "rgb(119, 158, 242)",
  dataModel: "rgb(235, 112, 146)",
  dashboard: "rgb(239, 110, 100)",
  genericLoading: "rgb(184, 195, 206)",
  numberInput: "rgb(150, 148, 146)",
  toggle: "rgb(150, 148, 146)",
  address: "rgb(75, 192, 118)",
  priceBooks: "rgb(4, 132, 75)",
  entityMilestone: "rgb(244, 151, 86)",
  wealthManagement: "rgb(0, 210, 190)",
  customers: "rgb(14, 181, 138)",
  story: "rgb(84, 105, 141)",
  serviceAppointment: "rgb(126, 139, 228)",
  maintenancePlan: "rgb(42, 115, 158)",
  dataIntegrationHub: "rgb(42, 115, 158)",
  workPlanTemplateEntry: "rgb(68, 148, 136)",
  hierarchy: "rgb(52, 190, 205)",
  customComponentTask: "rgb(138, 118, 240)",
  partnerMarketingBudget: "rgb(14, 181, 138)",
  assetActionSource: "rgb(49, 122, 146)",
  skillRequirement: "rgb(250, 151, 92)",
  filterCriteriaRule: "rgb(42, 115, 158)",
  location: "rgb(75, 192, 118)",
  radioButton: "rgb(150, 148, 146)",
  avatarLoading: "rgb(184, 195, 206)",
  article: "rgb(242, 207, 91)",
  invocableAction: "rgb(84, 105, 141)",
  device: "rgb(127, 141, 225)",
  proposition: "rgb(60, 151, 221)",
  snippets: "rgb(14, 181, 138)",
  customerPortalUsers: "rgb(60, 151, 219)",
  actionsAndButtons: "rgb(252, 185, 91)",
  recordUpdate: "rgb(235, 112, 146)",
  shiftType: "rgb(235, 112, 146)",
  logACall: "rgb(72, 195, 204)",
  quotes: "rgb(136, 198, 81)",
  questionFeed: "rgb(242, 207, 91)",
  kanban: "rgb(60, 151, 221)",
  workPlan: "rgb(68, 148, 136)",
  shiftTemplate: "rgb(235, 112, 146)",
  merge: "rgb(242, 207, 91)",
  expense: "rgb(58, 49, 128)",
  dataset: "rgb(176, 112, 230)",
  shiftPreference: "rgb(235, 112, 146)",
  productConsumed: "rgb(85, 188, 156)",
  canvas: "rgb(129, 153, 175)",
  forecasts: "rgb(107, 189, 110)",
  relationship: "rgb(60, 151, 221)",
  serviceResource: "rgb(126, 139, 228)",
  filter: "rgb(21, 57, 238)",
  salesPath: "rgb(42, 115, 158)",
  shiftSchedulingOperation: "rgb(235, 112, 146)",
  events: "rgb(60, 151, 219)",
  dataMapping: "rgb(6, 28, 63)",
  sms: "rgb(136, 198, 81)",
  rtcPresence: "rgb(71, 207, 210)",
  avatar: "rgb(98, 183, 237)",
  recordDelete: "rgb(235, 112, 146)",
  solution: "rgb(143, 201, 114)",
  agentHome: "rgb(0, 57, 107)",
  partnerFundClaim: "rgb(14, 181, 138)",
  individual: "rgb(60, 151, 221)",
  customNotification: "rgb(107, 183, 228)",
  dateInput: "rgb(150, 148, 146)",
  catalog: "rgb(2, 126, 70)",
  template: "rgb(60, 151, 221)",
  activationTarget: "rgb(220, 113, 209)",
  sellingModel: "rgb(173, 123, 238)",
  shipment: "rgb(126, 139, 228)",
  recordSignatureTask: "rgb(235, 112, 146)",
  event: "rgb(235, 112, 146)",
  insurance: "rgb(0, 210, 190)",
  liveChatVisitor: "rgb(246, 137, 96)",
  textarea: "rgb(150, 148, 146)",
  workStep: "rgb(68, 148, 136)",
  picklistType: "rgb(150, 148, 146)",
  survey: "rgb(49, 159, 214)",
  link: "rgb(122, 154, 230)",
  messagingSession: "rgb(52, 190, 205)",
  video: "rgb(176, 112, 230)",
  listEmail: "rgb(139, 174, 181)",
  recycleBin: "rgb(21, 137, 228)",
  document: "rgb(186, 172, 147)",
  productTransfer: "rgb(248, 137, 98)",
  instoreLocations: "rgb(84, 105, 141)",
  recent: "rgb(108, 161, 233)",
  practitionerRole: "rgb(235, 112, 146)",
  password: "rgb(150, 148, 146)",
  expenseReport: "rgb(58, 49, 128)",
  branchMerge: "rgb(233, 105, 110)",
  insights: "rgb(236, 148, 237)",
  dropbox: "rgb(82, 174, 249)",
  calculatedInsights: "rgb(84, 167, 248)",
  employeeJobPosition: "rgb(0, 161, 224)",
  file: "rgb(186, 172, 147)",
  currencyInput: "rgb(150, 148, 146)",
  variable: "rgb(84, 105, 141)",
  teamMember: "rgb(242, 207, 91)",
  pricingWorkspace: "rgb(4, 132, 75)",
  agentSession: "rgb(248, 137, 96)",
  guidanceCenter: "rgb(127, 141, 225)",
  groupLoading: "rgb(184, 195, 206)",
  lead: "rgb(248, 137, 98)",
  email: "rgb(149, 174, 197)",
  serviceContract: "rgb(138, 118, 240)",
  decision: "rgb(255, 154, 60)",
  durationDownscale: "rgb(21, 137, 238)",
  snippet: "rgb(160, 148, 237)",
  entitlementProcess: "rgb(126, 139, 228)",
  contactList: "rgb(160, 148, 237)",
  planogram: "rgb(60, 151, 221)",
  activations: "rgb(176, 112, 230)",
  schedulingPolicy: "rgb(235, 112, 146)",
  channelProgramHistory: "rgb(14, 181, 138)",
  questionBest: "rgb(242, 207, 91)",
  collectionVariable: "rgb(84, 105, 141)",
  procedureDetail: "rgb(52, 190, 205)",
  salesValue: "rgb(60, 151, 221)",
  knowledge: "rgb(236, 148, 237)",
  dateTime: "rgb(150, 148, 146)",
  category: "rgb(2, 126, 70)",
  maintenanceWorkRule: "rgb(42, 115, 158)",
  dataStreams: "rgb(239, 110, 100)",
  textbox: "rgb(150, 148, 146)",
  leadInsights: "rgb(34, 176, 230)",
  waits: "rgb(255, 154, 60)",
  concur: "rgb(76, 195, 199)",
  feed: "rgb(98, 183, 237)",
  sort: "rgb(21, 57, 238)",
  messagingConversation: "rgb(52, 190, 205)",
  serviceReport: "rgb(126, 139, 228)",
  iotContext: "rgb(42, 115, 158)",
  assetStatePeriod: "rgb(49, 122, 146)",
  call: "rgb(242, 207, 91)",
  collection: "rgb(242, 207, 91)",
  stage: "rgb(255, 154, 60)",
  salesCadence: "rgb(84, 105, 141)",
  productRequestLineItem: "rgb(136, 198, 81)",
  returnOrderLineItem: "rgb(0, 150, 136)",
  chart: "rgb(31, 202, 160)",
  quickText: "rgb(98, 183, 229)",
  home: "rgb(239, 126, 173)",
  sossession: "rgb(84, 105, 141)",
  bundleConfig: "rgb(21, 137, 238)",
  employeeAsset: "rgb(0, 161, 224)",
  processException: "rgb(245, 166, 35)",
  productWarrantyTerm: "rgb(183, 129, 211)",
  stageCollection: "rgb(255, 154, 60)",
  productRequest: "rgb(136, 198, 81)",
  logging: "rgb(0, 161, 223)",
  assignedResource: "rgb(69, 193, 115)",
  returnOrder: "rgb(0, 150, 136)",
  poll: "rgb(105, 155, 225)",
  household: "rgb(0, 175, 160)",
  workStepTemplate: "rgb(68, 148, 136)",
};
